telegrambots-abilities/src/main/java/org/telegram/abilitybots/api/db/BackupMap.java

```diff
@@ -0,0 +1,36 @@
+package org.telegram.abilitybots.api.db;
+
+import org.telegram.abilitybots.api.util.Pair;
+
+import java.util.*;
+
+public class BackupMap<K, V> extends AbstractCollection<Pair<K, V>> implements Collection<Pair<K, V>> {
```

Comment:
Make class final, I don't see a use case beyond recovery

Reply:
This is supposed to be an internal class used for backup and recovery; class should be package private.